### ISSUE TITLE
Unable to find time zone names America/St Barthelemy

### ISSUE THREAD

**Issue by monove** [sproutsocial/walltime-js#49]
Any ideas?

**Comment by mj1856** [comment]
You need an underscore where you have a space.

**Comment by monove** [comment]
Uncaught Error: Unable to find time zone named America/St_Barthelemy
(WallTime 0.2.0)

**Comment by jgable** [comment]
I think this is because the America/St_Barthelemy rule is really a [Link to America/Guadeloupe](https://github.com/eggert/tz/blob/5ea7a94f6a57161575cac3a9698265c684d19e6c/northamerica#L2932).  I don't think I ever implemented link situations.

**Comment by robmadden** [comment]
Whatever @jgable says is right, so just listen to him.

**Comment by mj1856** [comment]
Yep, that would do it as well.  I'd recommend supporting links.  Zones occasionally move to links upon new versions of tzdb.

**Comment by getaaron** [comment]
I agree with @robmadden

**Comment by monove** [comment]
But how would I get a link mapping?

**Comment by geedew** [comment]
@robmadden ...
![@robmadden](https://cloud.githubusercontent.com/assets/37583/12906618/d0410a68-cea7-11e5-9c33-943e59d259d4.gif)

**Comment by az-tock** [comment]
+1 @getaaron

**Comment by jgable** [comment]
Why do I feel like I'm being hazed?  You know what the real hazing was, when I had to write this monstrosity for handling timezones in the first place.

Shouldn't people be using http://momentjs.com/timezone/ by now?  They really do a much better job at this and they have like [a million tests](http://momentjs.com/timezone/tests/).

Man, and this library is written in CoffeeScript, talk about nobody else being able to maintain it.  I'm stuck with this thing for life.

**Comment by monove** [comment]
"Moment Timezone has no data for America/St Barthelemy"

**Comment by monove** [comment]
never mnd that, i missed the underscore

**Comment by mj1856** [comment]
@jgable - Despite jumping on the moment.js / moment-timezone bandwagon, I've also tried to give fair credit to alternatives like walltime (and a few others).  Believe me, I considered answering this issue with "just use moment instead" - but thought you guys would frown on that.  Would you prefer we consider walltime discontinued and move along?  Just askin'.  :)

**Comment by jgable** [comment]
I think it would be relatively clear from looking at the commits/pulse graph that this library isn't being maintained.  I'm no longer at Sprout Social so I can't speak for them, but I'd be +1 to putting a notice in the readme.

**Comment by soung3** [comment]
We're actually actively in the process of converting over to Moment.js at Sprout Social. We'll update the README to state the end of support, unless someone else wants to carry the torch.

**Comment by mj1856** [comment]
Thanks for clarifying.  If I can do anything to ease the transition, please let me know.
